### PR TITLE
Unbalanced join detector

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

If the round is unbalanced in favor of marines and the xenos still have a lot of larvas (more than 3), the game will warn player when they try to join as marine, asking them to consider joining as a xenos instead

## Why It's Good For The Game

Capture

## Changelog
:cl:
add: Popup when joining as a marine and xenos are losing
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
How about the reverse? Just in case that they're losing groundside?

**Comment by BraveMole** [comment]
I never saw a lack of marines

File: code/modules/mob/new_player/new_player.dm
Comment (SplinterGP) [inline code/modules/mob/new_player/new_player.dm:273]: Just use tgui_alert() instead.

**Review by SplinterGP — APPROVED** [review]

**Comment by SplinterGP** [comment]
MISSCLICK AGH DINDT MEAN TO APPROVE

File: code/modules/mob/new_player/new_player.dm
Comment (SplinterGP) [inline code/modules/mob/new_player/new_player.dm:41]: ditto use tgui_alert() instead if possible

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/mob/new_player/new_player.dm
Comment (BraveMole) [reply code/modules/mob/new_player/new_player.dm:41]: TOO LATE YOU APPROVED I WONT DO NOTHING AH

Comment (SplinterGP) [reply code/modules/mob/new_player/new_player.dm:41]: > TOO LATE YOU APPROVED I WONT DO NOTHING AH

SUFFER DO NOTHING NO MERGE

**Comment by BraveMole** [comment]
Waiting for tivi's ui to get merge

**Comment by SplinterGP** [comment]
> Waiting for tivi's ui to get merge

(y)

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-38). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**
